src/App.tsx

```diff
@@ -1,7 +1,8 @@
 import { Component } from 'preact'
 import { useState, useEffect } from 'preact/hooks'
 
-import Flipper from './Flipper'
+// import Flipper from './FlipperBK'
+import Flipper from './components/Flipper'
```

Comment:
Please avoid creating type-based folders like `models`, `components`. The preferred file organization for this project (and my projects) is “packlets” system &rarr; https://www.npmjs.com/package/@rushstack/eslint-plugin-packlets

For now I recommend just putting all files in the same directory, I can adjust the structure into packlets 😂.